### PR TITLE
Refactor Pub-API presenters

### PR DESCRIPTION
**Pull request by mgrassotti**
Refactor common methods in DetailedGuide and CaseStudy presenters

### CODE REVIEW

File: app/presenters/publishing_api_presenters/publication_helper.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module PublishingApiPresenters::PublicationHelper
```
Comment (boffbowsh) [inline app/presenters/publishing_api_presenters/publication_helper.rb]: I think this name might lead to confusion with Publications. Perhaps just move it into the `Edition` base presenter instead?

Comment (mgrassotti) [inline app/presenters/publishing_api_presenters/publication_helper.rb]: 👍

File: app/presenters/publishing_api_presenters/applicability_helper.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+module PublishingApiPresenters::ApplicabilityHelper
+private
+
+  def nation_to_sym(nation)
+    nation.tr(' ', '_').downcase.to_sym
```
Comment (boffbowsh) [inline app/presenters/publishing_api_presenters/applicability_helper.rb]: This can just be `nation.underscore.to_sym`.

**Comment by boffbowsh** [comment]
Small comment, otherwise 👍

File: app/presenters/publishing_api_presenters/applicability_helper.rb (hunk 2 above)
Comment (fofr) [inline app/presenters/publishing_api_presenters/applicability_helper.rb]: That's not correct @boffbowsh:

```
irb(main):001:0> "Northern Ireland".tr(' ', '_').downcase.to_sym
=> :northern_ireland
irb(main):002:0> "Northern Ireland".underscore.to_sym
=> :"northern ireland"
```

Comment (boffbowsh) [inline app/presenters/publishing_api_presenters/applicability_helper.rb]: :facepalm:.

**Comment by boffbowsh** [comment]
👍

File: app/presenters/publishing_api_presenters/political_helper.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+module PublishingApiPresenters::PoliticalHelper
+private
+
+  def government
+    gov = item.government
+    {
+      title: gov.name,
+      slug: gov.slug,
+      current: gov.current?
+    }
```
Comment (fofr) [inline app/presenters/publishing_api_presenters/political_helper.rb]: This line is also associated with something being political:
https://github.com/alphagov/whitehall/blob/99034225d518705c56fb6b42c1a8a273854c1d5b/app/presenters/publishing_api_presenters/detailed_guide.rb#L32

I feel like they should somehow be kept together.

Comment (mgrassotti) [inline app/presenters/publishing_api_presenters/political_helper.rb]: @fofr do you mean something like this?
https://github.com/alphagov/whitehall/blob/7c05142b175ae4a9586c8827e000d5a6743dbb0e/app/presenters/publishing_api_presenters/political_helper.rb#L13-L17

Comment (fofr) [inline app/presenters/publishing_api_presenters/political_helper.rb]: Should `political_details` also merge in the government parts, as they are essentially connected?

```
{
  political: item.political?,
  government: government
}
```

Comment (mgrassotti) [inline app/presenters/publishing_api_presenters/political_helper.rb]: 👍  I was thinking about that

**Comment by gpeng** [comment]
:+1:

**Pull request merged**